# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../partimage/hotfix-fs_xfs-__u64.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./src/client/fs/fs_xfs.h.orig	2006-09-28 15:02:20.000000000 +0200
+++ ./src/client/fs/fs_xfs.h	2006-09-28 16:07:57.000000000 +0200
@@ -150,13 +150,13 @@
 typedef	__uint64_t	xfs_dfiloff_t;	// block number in a file 
 typedef	__uint64_t	xfs_dfilblks_t;	// number of blocks in a file 
 
-typedef uint64_t __u64;
-typedef int64_t  __s64;
+//typedef uint64_t __u64;
+//typedef int64_t  __s64;
 
-typedef __u64	xfs_off_t;
+typedef uint64_t	xfs_off_t;
 //typedef __s32	xfs32_off_t;
-typedef __u64	xfs_ino_t;	// <inode> type 
-typedef __s64	xfs_daddr_t;	// <disk address> type 
+typedef uint64_t	xfs_ino_t;	// <inode> type 
+typedef int64_t	xfs_daddr_t;	// <disk address> type 
 typedef char *	xfs_caddr_t;	// <core address> type 
 typedef off_t	linux_off_t;
 //typedef __kernel_ino_t	linux_ino_t;
